### ISSUE TITLE
test

### ISSUE THREAD

**Issue by MichaelCong** [tanluren/mobilenetv3-yolov3#6]
Hello, thank you very much for your work. After training according to your method, the result of detection is very poor, the speed is only 10 fps. Yolov3 is much better than that. What's the reason?

**Comment by tanluren** [comment]
Hi ,which device did you test on ? the speed varies in different device. But it's at least 3x faster than darknet according to my test.

**Comment by MichaelCong** [comment]
My computer is configured as follows: I think it is because the GPU is not used up and it runs very slowly (CPU processing time).
Cpu: intel® CoreTM i7-6950X CPU @ 3.00GHz × 20
GPU: GeForce GTX 1080 Ti/PCIe/SSE2 x4

**Comment by tanluren** [comment]
Did you install tensorflow-gpu?

**Comment by MichaelCong** [comment]
I installed‘’pip install tensorflow-gpu‘’，But, I don't know why I can't use the GPU for training and testing. Thank you very much, can you help me?

**Comment by tanluren** [comment]
Could you train tensorflow model with GPU?

**Comment by MichaelCong** [comment]
NO, I don't know why I can't use the GPU for training and testing. Thank you very much。Is the problem of training code or the problem of my software installation?

**Comment by tanluren** [comment]
did you install cuda and cudnn correctly? did you install both tensorflow and tensorflow-gpu which lead to wrong default backend? could you run any other GPU code? keras will run on single gpu automatically if your installation is right ,if you want to use multi gpu you need to call multi_gpu_model api explicitly .In addition ,the model cant reach 10 fps on cpu. this model is very slow on first time inference due to session initializing, but it will be then fast if you keep the session live.

**Comment by MichaelCong** [comment]
Hello, I installed CUDA and cudnn correctly, and I can use pytorch framework to train correctly with multiple GPUs.
When I first used TensorFlow, I couldn't use GPU. Maybe I installed TensorFlow and TensorFlow-GPU, which caused conflicts. I'm trying to adjust it. Thank you very much.

**Comment by zihaozhang9** [comment]
Input_size=416 . So the mobilenet model may be larger than in the paper.

> Hello, thank you very much for your work. After training according to your method, the result of detection is very poor, the speed is only 10 fps. Yolov3 is much better than that. What's the reason?
